File: C#/BruteForce.cs
using System;

namespace BruteForce {
    public class  Program {
        static bool IsPrime(int num) {
            if (num <= 1) return false;
            if (num == 2) return true;
            for (int i = 2; i<num; i++) {
                if (num%i == 0)
                    return false;
            }
            return true;
        }

        static void Main(string[] args) {
            Console.WriteLine("Enter a number to check if it's prime or composite");
            int num = Int32.Parse(Console.ReadLine());
            Console.WriteLine("The number you entered is "+(IsPrime(num)?"prime":"not prime"));
        }
    }
}

File: PHP/BruteForce.php
<?php
function isPrime($num) {
    if ($num <= 1) return false;
    if ($num == 2) return true;
    for ($i = 2; $i < $num; $i++) {
        if ($num%$i == 0) return false;
    }
    return true;
}

$num = 10;
echo "The number is ";
if (!isPrime($num)) echo "not ";
echo "prime.\n";
?>
